### DISCUSSION TITLE
Apakah AdblockID mendukung pi-hole atau pemblokir berbasis DNS?

### DISCUSSION THREAD

**Discussion by arfshl** [realodix/AdBlockID#1111] [Q&A] [answered]
Apakah AdblockID mendukung pi-hole atau pemblokir berbasis DNS?

**Comment by realodix** [comment, marked as answer]
Tidak.

Sebagai gantinya Anda bisa menggunakan [Energized](https://github.com/EnergizedProtection/block), di bagian regional mereka menyediakan AdBlockID.

**Reply by arfshl** [reply]
Terimakasih atas jawabannya ini sangat membantu
